docs/helpers.md

```diff
@@ -110,7 +112,7 @@ module.exports = MyHelper;
 
 ## Puppeteer Example
 
-Puppteer has [nice and elegant API](https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md) which you can use inside helpers. Access `page` instance via `this.helpers.Puppeteer.page` from inside a helper.
+Puppteer has [nice and elegant API](https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md) which you can be used inside helpers. Access `page` instance via `this.helpers.Puppeteer.page` from inside a helper.
```

Comment:
`which you can be used inside helpers` ?
What the?
maybe `which can be used` or `which you can use`